plugins/experimental/magick/magick.cc

```diff
@@ -226,19 +226,17 @@ verify(const byte *const msg, const size_t mlen, const byte *const sig, const si
     }
   }
 
-  {
-    const int rc = EVP_DigestVerifyUpdate(evp.context, msg, mlen);
-    assert(1 == rc);
-    if (1 != rc) {
-      return false;
-    }
-  }
-
   ERR_clear_error();
 
   {
-    const int rc = EVP_DigestVerifyFinal(evp.context, sig, slen);
-    return 1 == rc;
+    const int rc = EVP_DigestVerify(evp.context, sig, slen, msg, mlen);
+    if (1 == rc) {
+      return true;
+    } else if (0 == rc) {
```

Comment:
If we check this case let's print debug log.

Reply:
Is my use of `Dbg` for this purpose good? I wonder whether we should do this for the API call to load the RSA certificate as well. The current implementation does not even handle such a failure in production. It continues to operate without the cert, and will probably cause everything to fail to verify, if it doesn't do something worse.

Reply:
It does debug asserts and it's probably assuming that it should never fail to load its RSA certificate.

Reply:
It just looked like doing exactly same thing as the `else` clause doesn't make sense. Current code still makes me wonder why the error is silently ignored in this case. Leaving a note/comment on the source code may be enough. I'm also fine with letting the `else` clause handle alll the error case.